doc/api/fs.md

```diff
@@ -4288,7 +4290,9 @@ arguments upon success.
 
 The optional `options` argument can be an integer specifying mode (permission
 and sticky bits), or an object with a `mode` property and a `recursive`
-property indicating whether parent folders should be created.
+property indicating whether parent folders should be created. Calling `fs.mkdir`
+with a path to a directory that exists results in an error an error only
```

Comment:
```suggestion
with a path to a directory that exists results in an error only
```

Reply:
```suggestion
with a `path` to a directory that exists results in an error only
```

Reply:
Is "results in an error" the right phrasing for a promise-based API?